setup.cfg

```diff
@@ -8,4 +8,5 @@
 [flake8]
 ignore = E501,E127,E128,E265,E301,E302,E266,E731,N803,N806
 max-line-length = 100
-exclude = .git,.ropeproject,.tox,docs,setup.py,env,venv,models/migrations,tests/*,api_tests/*,scripts/*,src,website/uploads/,website/static/**/*,website/settings/*,framework/forms/*,addons/*/tests/*,env,venv,node_modules,admin/static/*,osf/migrations
+exclude = .git,.ropeproject,.tox,docs,setup.py,env,venv,models/migrations,tests/*,scripts/*,src,website/uploads/,website/static/**/*,website/settings/*,framework/forms/*,addons/*/tests/*,env,venv,node_modules,admin/static/*,osf/migrations
```

Comment:
Well this PR looks like it was a lot of fun.  Nice work @chennan47 - it will be good to have flake8 on the api_tests.